engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala

```diff
@@ -1,7 +1,8 @@
-package org.enso.compiler.codegen
+package org.enso.interpreter.runtime
```

Comment:
`IrToTruffle` moves next to `TruffleCompilerContext` implementation as it is _runtime related_ - not _compile (e.g. IR processing) time_ related activity.